stackstorm-nexus3/tests/test_run.py

```diff
@@ -0,0 +1,12 @@
+from st2tests.base import BaseActionTestCase
+
+from run import ActionManager
+from lib.exception import *
+
+class RunTestCase(BaseActionTestCase):
+    action_cls = ActionManager
+
+    def test_run_required_params(self):
+        action = self.get_action_instance(config={'config_profile': 'dev'})
+        #self.assertRaises(MissingParameterError,action.run)
```

Comment:
Why is this commented out?

But thank you for including at least one unit test.

Reply:
sorry @blag . I am new to testing and learning mock n fixtures. I am writing better tests cases and this will all be fixed. 

You are very polite in your review. really appreciate that. Honestly, i was worried how my first contribution will be taken by experts.